packages/p2p/src/components/my-ads/form-ads.jsx

```diff
@@ -227,7 +253,13 @@ class FormAds extends Component {
                                         </Field>
                                     </ThemedScrollbars>
                                     <FooterActions has_border>
-                                        <Button className='p2p-my-ads__form-button' secondary large type='reset'>{localize('Cancel')}</Button>
+                                        {this.state.error_message &&
+                                        <div className='p2p-my-ads__form-error'>
+                                            <Icon icon='IcAlertDanger' />
+                                            <p>{this.state.error_message}</p>
+                                        </div>
+                                        }
+                                        <Button className='p2p-my-ads__form-button' secondary large onClick={ () => this.props.handleShowForm(false) }>{localize('Cancel')}</Button>
```

Comment:
type="submit" here I think:

```suggestion
                                        <Button className='p2p-my-ads__form-button'  type="submit" secondary large onClick={ () => this.props.handleShowForm(false) }>{localize('Cancel')}</Button>
```

Reply:
buttons are type submit by default, do we want to be explicit?

Reply:
@negar-binary that's true I forgot about that 👍